src/extension.ts

```
@@ -425,6 +616,214 @@ console.log('Roo: After registering runserver command');
+						});
+						return;
+					case 'addProxy':
+						try {
+							if (!proxyPoolManager) {
+								throw new Error('Proxy pool manager not initialized');
+							}
+							const proxyId = await proxyPoolManager.addProxy(message.url);
+							panel.webview.postMessage({
+								command: "addProxy(response)",
+								success: true,
+								proxyId: proxyId,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)